lib/3scale_toolbox/commands/import_command/openapi/threescale_api_spec.rb

```diff
@@ -5,9 +5,10 @@ module OpenAPI
         class ThreeScaleApiSpec
           attr_reader :openapi
 
-          def initialize(openapi, base_path = nil)
+          def initialize(openapi, base_path = nil, prefix_matching = false)
```

Comment:
In case this approach is taken, is it a good idea to have two default parameters? If we were to have the case where we want to set prefix_matching but not base_path(we want default for it) making this would force us to define the base_path parameter explicitly as nil.

Reply:
I think it is ok.